src/context.h

```diff
@@ -202,6 +202,13 @@ struct ly_ctx;
                                         'require-instance false;'. It also enables usage of
                                         [lyd_leafref_get_links](@ref lyd_leafref_get_links) and
                                         [lyd_leafref_link_node_tree](@ref lyd_leafref_link_node_tree) APIs. */
+#define LY_CTX_BASIC_PLUGINS_ONLY 0x0800 /**< By default, context loads all available built-in plugins. This options prohibits
+                                        loading of advanced built-in plugin types (ipv4-address, ipv6-address, etc.). Instead the value is
+                                        processed by basic built-in pluings (usually string) with all its behavioral implications as
+                                        hex-string comparison becoming case-sensitive.
+                                        Change of this flag during the lifetime of context is not fully supported. Once the advanced
+                                        plugins are loaded during context creation, they will be used regardless of this flag.
+                                        Therefore it is recommended to set this flag during the first context creation. */
```

Comment:
Clarify this a bit and please use built-in as in "built-in YANG types" and not "built-in libyang plugins". Then the option can be renamed to `LY_CTX_BUILTIN_TYPE_PLUGINS_ONLY` which should give a better idea about the loaded plugins.

Reply:
Done, but used `LY_CTX_BUILTIN_PLUGINS_ONLY` as it disables also built-in extensions